City-of-Helsinki/hel-fi-drupal-grants
# UHF-10349: Restructuring code

# [UHF-10349](https://helsinkisolutionoffice.atlassian.net/browse/UHF-10349)
Move conversion functions from GrantsHandler into their own helper class.

## What was done

* Added new class `ConversionHelpers`
* Moved conversion functions from `GrantsHandler` to `ConversionHelpers`
* Fixed all references to moved functions

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout UHF-10349`
  * `make fresh`
* Run `make drush-cr`

## How to test
* Try to produce any calls to moved `GrantsHandler` functions using grep or similar (you shouldn't find any)
* Try filling a couple of applications and sending them
* If you encounter any errors, check `docker compose logs app -f|grep error` to see what is up

## Designers review


* [] This PR has been visually reviewed by a designer (Name of the designer)


## Automatic- / Regression tests



[UHF-10349]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-10349?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
